models/model_runner.py

```diff
@@ -26,14 +26,31 @@ def get_args():
     return parser.parse_args()
 
 
-def create_optional_args() -> List:
+def create_reid_opts() -> List:
     """
-    Creates a list of optional arguments that if given should be appended to the sys.call arguments.
+    Creates a list of optional arguments that should be passed to the reid model through the `--reid_opts` param.
     """
-    optional_args: List = []
+    reid_opts: List = []
     if args.reid_opts:
         for opt in args.reid_opts:
-            optional_args.append(opt)
+            reid_opts.append(opt)
+    reid_opts.extend(['MODEL.DEVICE', args.device])
+    return reid_opts
+
+
+def create_optional_args() -> List:
```

Comment:
How do these optional args work along side the Reid optional args? Add explanation in model runner init?

Reply:
The new optional args I added are for the tracking model mainly. The Reid optional args are derived from the `--reid_opts` arg and they are separate from the other args.